### DISCUSSION TITLE
13.0.0-2 version exists?

### DISCUSSION THREAD

**Discussion by azinsharaf** [BurntSushi/ripgrep#2304] [General] [answered]
Hi, as of today (Sep 19 2022) according to GitHub Release page the `ripgrep` stable version is `13.0.0` but why `sudo apt upgrade` in Ubuntu wants to upgrade it to `13.0.0-2`? Does it exist? I don't see it in the release page.

**Comment by BurntSushi** [comment, marked as answer]
The latest version of ripgrep is 13.0.0.

I don't know what Ubuntu does or why it's showing ripgrep 13.0.1. This is more a question for Ubuntu people.

**Comment by azinsharaf** [comment]
actually it says `13.0.0-2 amd 64`.  edited the title. Not sure what `-2` means.

**Reply by BurntSushi** [reply]
Also an Ubuntu question, but usually that just means the distro made some change to the package itself. So the version number is something like `{upstream-version}-{distro-version}`, where `distro-version` is just an auto-incrementing integer that is reset to `1` every time `upstream-version` changes.
